selectors/views/channel_sidebar.ts

```diff
@@ -27,9 +27,12 @@ import {memoizeResult} from 'mattermost-redux/utils/helpers';
 
 import {DraggingState, GlobalState} from 'types/store';
 
-export function isUnreadFilterEnabled(state: GlobalState): boolean {
-    return state.views.channelSidebar.unreadFilterEnabled && !shouldShowUnreadsCategory(state);
-}
+export const isUnreadFilterEnabled = createSelector(
```

Comment:
Same here since the input selectors just return booleans

Reply:
reverted